### ISSUE TITLE
iOS 11.3 Crash - Warning: merge shape is not supported.

### ISSUE THREAD

**Issue by jwebbnelson** [react-community/lottie-react-native#309]
I have 3 animations that have been implemented in an iOS/Android application for over a month of testing without a single problem. 

**After updating to Xcode 9.3 and iOS 11.3,** the application now crashes with the warning,
 _"+[LOTShapeGroup shapeItemWithJSON:]: Warning: merge shape is not supported. name: Merge Paths 1"_, repeated a dozen times. 

The animations still work as intended on Android and this is the first mention of the use of Merge Paths in my animations or any bug in Lottie implementation. Is this a known problem for the new version of iOS or should one anticipate critical crashes following a major OS update?

**Comment by x697** [comment]
I'm getting this error too. Looked around and saw that merge paths are not supported, but I'm not using merge paths.

**Comment by jwebbnelson** [comment]
@x697  Neither am I. I've tried re-exporting new JSON files from Bodymovin (limited to one layer) and still get the error

**Comment by JakeRawr** [comment]
Not sure if it's related. My lottie files run on xcode simulator with ios 9.3. But When I run it on xcode simulator 11.4, it shows black screen instead.

**Comment by jwebbnelson** [comment]
I was able to get my own animations working by returning to the After Effects file, expanding the contents of each layer, and removing any merge path and associated group. If I only removed the merge path, my layer's fill wouldn't look as expected. 

Once I removed the merge paths and re-exported the json file using BodyMovin, the animations worked in my iOS application but I still get a warning at runtime in the console regarding merge paths. Android didn't require me to update my animations and hasn't crashed. 

I'm not sure if my particular scenario helps anyone else. I'm familiar with the merge path rules, but the fact that my iOS application was able to run the animations initially without a problem is strange. I'm now on iOS 11.4 and the animations perform as expected.
